egui_web/src/webgl1.rs

```diff
@@ -196,11 +205,16 @@ impl WebGlPainter {
 
         let mut positions: Vec<f32> = Vec::with_capacity(2 * mesh.vertices.len());
         let mut tex_coords: Vec<f32> = Vec::with_capacity(2 * mesh.vertices.len());
+        let is_offscreen = self.get_is_offsceen(mesh.texture_id);
         for v in &mesh.vertices {
             positions.push(v.pos.x);
             positions.push(v.pos.y);
             tex_coords.push(v.uv.x);
-            tex_coords.push(v.uv.y);
+            if let Some(true) = is_offscreen {
+                tex_coords.push(1.0 - v.uv.y);
+            } else {
+                tex_coords.push(v.uv.y);
+            }
```

Comment:
Why store the egui texture in another direction from other textures? How are you loading these other images that makes them appear upside-down?

Also, do you know you can flip images with https://docs.rs/egui/0.10.0/egui/widgets/struct.Image.html#method.uv ?

Reply:
okay i didnt see it.
 remove off screen flag but register function still alive